### DISCUSSION TITLE
How to make @screen not prompt warning in scss

### DISCUSSION THREAD

**Discussion by lxccc812** [unocss/unocss#3375] [Q&A] [answered]
For example this
![image](https://github.com/unocss/unocss/assets/110974969/f5c35bf7-7cef-4fa1-bad7-e5a85bd61249)

**Comment by Shyam-Chen** [comment]
Configure VS Code to ignore `unknownAtRules`:

```ts
  "scss.lint.unknownAtRules": "ignore"
```

**Reply by lxccc812** [reply]
Which configuration file needs to be added? .eslintrc.js does not seem to work.

![image](https://github.com/unocss/unocss/assets/110974969/5084c8e8-c304-403e-bf42-8cb1590b86c2)

**Reply by Shyam-Chen** [reply, marked as answer]
It's the VS Code configuration file (`settings.json`), not ESLint.
